pos_environment/models/pos_config.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from openerp.osv import fields as oldfields
+from openerp import models
+from openerp.addons.server_environment import serv_config
+
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    _columns = {
```

Comment:
Is there a reason to have an old api syntax for a V9 serie ?

I think that the following line should work.

```

from openerp import models
[...]
proxy_ip = fields.Char(compute=lambda s: s._compute_proxy_ip())

```

Reply:
Indeed, this should be changed as suggested.